lib/matplotlib/markers.py

```diff
@@ -143,6 +143,17 @@
 
 
 class MarkerStyle:
+    """
+    A class representing marker types.
+
+    Attributes
+    ----------
+    markers : list of known markers
+
+    fillstyles : list of known fillstyles
+
+    filled_markers : list of known filled markers
```

Comment:
First line really should be the type, so:
```suggestion
    markers : list
        All known markers.
    fillstyles : list
        All known fillstyles.
    filled_markers : list
        All known filled markers.
```